common/models/acl.js

```diff
@@ -378,6 +379,18 @@ module.exports = function(ACL) {
   };
 
   /**
+   * Test if ACL permission is ALLOW
+   * @returns {Boolean} true if ACL permission is ALLOW
+   */
+  ACL.prototype.isAllowed = function(defaultPermission) {
+    var permission = this.permission;
+    if (permission === ACL.DEFAULT) {
```

Comment:
I assumed we shouldn't treat the case where ACL.permission is not set as equivalent to ACL.permission = ACL.DEFAULT. 

Currently ([see here](https://github.com/ebarault/loopback/blob/0a2d3f9dc6ef2eff2fbab3d734cdb4d159e5b32b/common/models/acl.js#L488)) ACL.permission = undefined is not handled as equivalent to DEFAULT), and thus [this conditional test](https://github.com/ebarault/loopback/blob/0a2d3f9dc6ef2eff2fbab3d734cdb4d159e5b32b/common/models/acl.js#L494) would always fail if an ACL defined by code does not define the permission property to either DEFAULT, DENY or ALLOW

(did not verify for ACLs defined through the model JSON definition : how are handled missing `permission` property?)